h/models/organization.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+
+import sqlalchemy as sa
+import slugify
+
+from h.db import Base
+from h.db import mixins
+
+
+ORGANIZATION_NAME_MIN_CHARS = 1
+ORGANIZATION_NAME_MAX_CHARS = 25
+ORGANIZATION_LOGO_MAX_CHARS = 10000
+
+
+class Organization(Base, mixins.Timestamps):
+    __tablename__ = 'organization'
+
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+
+    name = sa.Column(sa.UnicodeText(), nullable=False, index=True)
+
+    logo = sa.Column(sa.UnicodeText())
+
+    def __init__(self, **kwargs):
+        super(Organization, self).__init__(**kwargs)
+
+    @property
+    def slug(self):
+        """A version of this organization's name suitable for use in a URL."""
+        return slugify.slugify(self.name)
```

Comment:
Do organizations need to have slugs? Since we aren't going to have pages for organizations (not yet anyway) I don't know if we have a use for organization slugs and we don't know whether we _will_ need them or not if and when we do have organization pages. Maybe just remove this method?

Reply:
(`slug` is used in the `__repr__()` method below but I think that could just use `name` instead)

Reply:
I don't think we absolutely need it right now or prob for quite sometime in the future, I agree, was more trying to be consistent with the other repr method.

Reply:
Did you two come to an agreement on this?

I note that slugs exist for use in URLs and before we can generate URLs for organizations we would need to decide what kind of identifier they would use - a pubid like groups? a numeric ID? a username-like value? If we picked the last option then the slug would no longer be needed.